src/components/DashboardSpeakers/DashboardSpeakersPresentation.js

```diff
@@ -23,48 +23,52 @@ function DashboardSpeakersPresentation({
         headerContent={headerContent}
         site={site}
       >
-        <DashboardTable.Presentation
-          queryResponse={infiniteQueryResponse}
-          title="Speakers"
-          tableHead={
-            <tr>
-              <th scope="col" className={tableHeaderClass}>
-                Name
-              </th>
-              <th scope="col" className={tableHeaderClass}>
-                Bio
-              </th>
-              {/* `relative` is added here due to a weird bug in Safari that causes `sr-only` headings to introduce overflow on the body on mobile. */}
-              <th scope="col" className={`relative ${tableHeaderClass}`}>
-                <span className="sr-only">Edit speaker</span>
-              </th>
-            </tr>
-          }
-          tableBody={infiniteQueryResponse?.data?.pages?.map((page) => (
-            <Fragment key={page.pageNumber}>
-              {page.results.map((speaker) => (
-                <tr key={speaker.id}>
-                  <td className="px-6 py-4 whitespace-normal text-sm font-medium text-charcoal-900">
-                    {speaker.name}
-                  </td>
-                  <td className="px-6 py-4 whitespace-normal text-sm text-charcoal-900">
-                    {speaker?.bio || '-'}
-                  </td>
-                  <td className="px-1 py-4 whitespace-nowrap text-right text-sm font-medium">
-                    <Link
-                      data-testid={`edit-speaker-${speaker.name}`}
-                      to={`/${site?.sitename}/dashboard/edit/speaker?id=${speaker?.id}`}
-                      className="btn-tertiary btn-md-icon mr-6"
-                    >
-                      {getIcon('Pencil')}
-                    </Link>
-                  </td>
-                </tr>
-              ))}
+        {infiniteQueryResponse?.hasResults && (
```

Comment:
Is the addition of this conditional necessary? This will prevent the InfiniteLoadBtn from rendering if there are no results and as such the "no results" message will not display.